tests/codestyle/fixtures/yodaStyle/actual.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if (null === $a) {
```

Comment:
Maybe `==`,`>=`, `<=`, `>`, `<` also should be added to tests?

Reply:
added more cases.